### DISCUSSION TITLE
not able to download

### DISCUSSION THREAD

**Discussion by beaucrtr** [RogueEdit/onlineRogueEditor#126] [General]
hey im not able to download anything the tab you mentioned in your YouTube video isn't there for me.

**Comment by FaultyDev** [comment]
yeah, i can't find the download either

**Comment by JulianStiebler** [comment]
We do not have any YT VIdeo ourselfs. we didnt made a single one.

But you can download the repo as usual. You found the discussions tab so you should know how to download code.
We removed releases because it was just annoying, the binarys are now just on the main branch and we may not provide any in the future, just the code.

**Comment by Noleary992** [comment]
so uhh, defender says that theres a virus detected
